source/concurrent/Promise.ooc

```diff
@@ -28,7 +28,7 @@ Promise: abstract class {
 		collector
 	}
 	kean_concurrent_promise_wait: unmangled func { this wait() }
-	kean_concurrent_promise_wait_timeout: unmangled func (timeout: Double) { this wait(timeout) }
+	kean_concurrent_promise_waitWithTimeout: unmangled func (timeout: Double) { this wait(timeout) }
```

Comment:
because it is more consistent with other functions in the public header (I can't give specific examples here :) )